Documentation/subcommands/api-service.md

```diff
@@ -7,6 +7,8 @@ The API service is implemented with [gRPC][grpc].
 The API service is designed to run without root privileges, and currently provides a read-only interface.
 The API service is optional for running pods, the start/stop/crash of the API service won't affect any pods or images.
 
+![api-service](api-service.svg)
```

Comment:
I would do the image slightly smaller on [here](https://github.com/kinvolk/rkt/blob/b90939f9549bab3151766be504f36581ebf785d8/Documentation/subcommands/api-service.md#overview) too.